study-builder/tenants/rgp/emails/change_password.html

```diff
@@ -0,0 +1,127 @@
+            line-height: 100%;
+        }
+
+        #outlook a {
+            padding: 0;
+        }
+
+        .ExternalClass,
+        .ReadMsgBody {
+            width: 100%;
+        }
+
+        a,
+        blockquote,
+        body,
+        li,
+        p,
+        table,
+        td {
+            -webkit-text-size-adjust: 100%;
+            -ms-text-size-adjust: 100%;
+        }
+
+        table,
+        td {
+            mso-table-lspace: 0;
+            mso-table-rspace: 0;
+        }
+
+        img {
+            -ms-interpolation-mode: bicubic;
+            border: 0;
+            height: auto;
+            outline: 0;
+            text-decoration: none;
+        }
+
+        table {
+            border-collapse: collapse !important;
+        }
+
+        body {
+            height: 100%!important;
+            margin: 0;
+            padding: 0;
+            font-family: Roboto, sans-serif;
+        }
+
+        @font-face {
+            font-family: Roboto, sans-serif;
+            src: url(https://fonts.googleapis.com/css?family=Roboto:300,500,600);
+            font-weight: 400;
+            font-style: normal;
+        }
+    </style>
+</head>
+
+<body style="background: #f5f5f5; width: 100%; height: 100%;">
+<!-- Translate the email into the appropriate language -->
+{% if user.user_metadata.language == "es" %}
+{% assign heading = "[es]Password Change Request" %}
+{% assign p1 = "[es]You have submitted a password change request for your Rare Genomes Project account." %}
+{% assign p2 = "[es]Click the button below to confirm and choose a new password." %}
+{% assign p3 = "[es]Reset my password" %}
+{% assign p4 = "[es]Sincerely," %}
+{% assign p5 = "[es]The Rare Genomes Project Team" %}
+{% assign p6 = "[es]If you did not make this request, please contact us." %}
+{% else %}
+{% assign heading = "Password Change Request" %}
+{% assign p1 = "You have submitted a password change request for your Rare Genomes Project account." %}
+{% assign p2 = "Click the button below to confirm and choose a new password." %}
+{% assign p3 = "Reset my password" %}
+{% assign p4 = "Sincerely," %}
+{% assign p5 = "The Rare Genomes Project Team" %}
+{% assign p6 = "If you did not make this request, please contact us." %}
+{% endif %}
+
+<center>
+    <table style="background: #ffffff; width: 600px; margin-top: 50px; padding: 0; -webkit-text-size-adjust: 100%;-ms-text-size-adjust: 100%; mso-table-lspace: 0pt; mso-table-rspace: 0pt; font-family: Roboto, sans-serif; border-collapse: collapse !important;"
+           align="center" border="0" cellpadding="0" cellspacing="0" width="100%">
+        <tr style="width: 600px;">
+            <td align="left" style="margin: 0; padding: 40px 0 40px 50px; font-family: Roboto, sans-serif; width: 175px;">
+                <img src="https://image.ibb.co/hSFqi5/logo.jpg" alt="Rare Genomes Project logo" style="-ms-interpolation-mode: bicubic;border: 0;height: auto; outline: none;text-decoration: none;">
```

Comment:
Who's hosting this logo right now? Will it be better for us to host it so we ensure this URL won't be broken?

Reply:
Thanks for catching this!